releasewarrior/helpers.py

```diff
@@ -39,7 +39,10 @@ def get_incomplete_releases():
     incomplete_releases = {}
     for abs_release_file, release_data in get_current_releases().items():
         if not all(release_data["builds"][-1]["human_tasks"].values()):
-            # this release is complete!
+            if release_data["builds"][-1].get("aborted"):
```

Comment:
we will need to address this in get_complete_releases() too. otherwise, postmortem call will never clean it up.

maybe we should have a:

```python
def is_complete(release_data):
    return all(release_data["builds"][-1]["human_tasks"].values()) or release_data["builds"][-1].get("aborted")
```

and use that in both get_incomplete_releases and get_incomplete_releases ?
    all(release_data["builds"][-1]["human_tasks"].values())